.github/workflows/dynamoMSbuild.yml

```diff
@@ -0,0 +1,28 @@
+# Build Dynamo using latest VS and DotNET
+name: Dynamo-VS2019Build
+on: [push]
+jobs:
+ build:
+  runs-on: windows-latest
+  steps:
+    - name: Checkout Dynamo Repo
+      uses: actions/checkout@v2
+      with:
+        path: Dynamo
+        repository: DynamoDS/Dynamo
+    - name: Setup Nuget.exe to use on VM
+      uses: nuget/setup-nuget@v1
+    - name: Nuget Restore in Dynamo solution
+      run: nuget restore $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln
+    - name: Build Dynamo with MSBuild
+      run: |
+        echo "***Execute ls, and you should see two folders created***"
+        ls
+        echo "***Continue with the build, Good luck developer!***"
+        cd "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\"
+           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln
+    # look for Dynamo
+    - name: Find Dynamo
+      run: |
```

Comment:
@QilongTang will this actually fail if Dynamo binaries are not created? Can we assert anything here?

Reply:
Not free, going to use https://github.com/marketplace/actions/file-existence

Reply:
@QilongTang if thats too difficult, what about just tryin to execute a DynamoCLI.exe instance?

Reply:
Yes it will fail because bash will fail

Reply:
@mjkkirschner To clarify, end up using bash only to reduce complexity